src/styles/normalize.css

```diff
@@ -0,0 +1,16 @@
+/* #region normalize */
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+
+  --font-family-regular: 400;
+  --font-family-bolt: 700;
+  --main-accent: #060b35;
+  --secondary-accent: #616070;
+  --white-accent: #fff;
+  --blue-accent: #00acdc;
+}
+
```

Comment:
From the videos, I saw that we need to make variables.css file with all variables in it. Now you have them in normaliza.css file. I don`t know if it is wrong, but need to comment:)
Just use pseudoclass :root {}